src/metatrain/utils/io.py

```diff
@@ -168,7 +173,7 @@ def load_model(
         architecture = import_architecture(architecture_name)
 
         try:
-            return architecture.__model__.load_checkpoint(path)
+            return architecture.__model__.load_checkpoint(path, context="export")
```

Comment:
Would be nice to create a test that `load_model` is loading the `export` context from a checkpoint.

Maybe we can use a pet checkpoint where we know that that weighs are different for export and restart?